packages/react/src/components/EmptyState/EmptyState.jsx

```diff
@@ -72,59 +72,61 @@ const defaultProps = {
  * Component to set empty states
  * For reference, visit https://pages.github.ibm.com/ai-applications/design/components/empty-states/usage/
  */
-const EmptyState = ({ title, icon, body, action, secondaryAction, className, testId, testID }) => (
-  <div
-    className={classnames(`${iotPrefix}--empty-state`, className)}
-    // TODO: remove deprecated testID in v3.
-    data-testid={testID || testId}
-  >
-    <div className={`${iotPrefix}--empty-state--content`}>
-      {icon &&
-        React.createElement(typeof icon === 'string' ? icons[icon] : icon, {
-          className: `${iotPrefix}--empty-state--icon`,
-          alt: '',
-          'data-testid': `${testID || testId}-icon`,
-        })}
-      <h3
-        className={`${iotPrefix}--empty-state--title`}
-        // TODO: remove deprecated testID in v3.
-        data-testid={`${testID || testId}-title`}
-      >
-        {title}
-      </h3>
-      <p
-        className={`${iotPrefix}--empty-state--text`}
-        // TODO: remove deprecated testID in v3.
-        data-testid={`${testID || testId}-body`}
-      >
-        {body}
-      </p>
-      {action && (
-        <div
-          className={`${iotPrefix}--empty-state--action`}
+const EmptyState = ({ title, icon, body, action, secondaryAction, className, testId, testID }) => {
+  return (
+    <div
+      className={classnames(`${iotPrefix}--empty-state`, className)}
+      // TODO: remove deprecated testID in v3.
+      data-testid={testID || testId}
+    >
+      <div className={`${iotPrefix}--empty-state--content`}>
+        {icon &&
+          React.createElement(typeof icon === 'string' ? icons[icon] : icon, {
+            className: `${iotPrefix}--empty-state--icon`,
+            alt: '',
+            'data-testid': `${testID || testId}-icon`,
+          })}
+        <h3
+          className={`${iotPrefix}--empty-state--title`}
           // TODO: remove deprecated testID in v3.
-          data-testid={`${testID || testId}-action`}
```

Comment:
This change set is off. I did not make these changes.

Reply:
I think this is just github being weird about the whitespace changes. If you view it with "ignore whitespace changes" on it looks fine.

Reply:
weird. Looking at next branch and this file doesn't have this code either. It appears to be right. https://github.com/carbon-design-system/carbon-addons-iot-react/blob/b9b9ce73515223a1fa7082b5afaebad8b786af92/packages/react/src/components/EmptyState/EmptyState.jsx#L91

Reply:
That's _very_ strange. Github really didn't like you adding that return statement....

Reply:
whitespace diffing is hard apparently.